### DISCUSSION TITLE
How to use specific GPU with ott-jax

### DISCUSSION THREAD

**Discussion by prabhant** [ott-jax/ott#330] [Q&A]
Hi,

How can i use GPU_ID example: 7 in ott-jax. 
Using it directly via jax.device does not work. CUDA_VISIBLE_DEVICES also does not work. OTT-JAX ends up using only gpuid 0 every time.

**Comment by michalk8** [comment]
Hi @prabhant ,
setting, e.g., `CUDA_VISIBLE_DEVICES='1'` before start of the program should be working as expected. What `jax/ott-jax` versions are you using?
